test/Amazon.SecretsManager.Extensions.Caching.Tests/IntegrationTests.cs

```diff
@@ -4,35 +4,36 @@
     using Amazon.SecretsManager.Model;
     using System;
     using System.Threading;
+    using System.Threading.Tasks;
     using System.Collections.Generic;
     using System.IO;
     using System.Linq;
 
     // Performs test secret cleanup before and after integ tests are run
-    public class TestBase : IDisposable
+    public class TestBase : IAsyncLifetime
     {
-        public static IAmazonSecretsManager Client = new AmazonSecretsManagerClient();
+        public static IAmazonSecretsManager Client = new AmazonSecretsManagerClient(Amazon.RegionEndpoint.USWest2);
```

Comment:
Depending on the environments these tests are run in, the default region is not always picked up correctly. Made it explicit.